tcms/rpc/tests/test_testrun.py

```diff
@@ -409,6 +409,11 @@ def _fixture_setup(self):
             "build": self.build.pk,
             "summary": "TR created",
             "manager": UserFactory().pk,
+            "start_date": "2020-05-01 00:00:00",
```

Comment:
@atodorov did I understand right your last comment? Are these the changes that you suggested? You are right, the tests are failing now with the following change.

Reply:
From what I can tell the changes in the last commit look good, need to see the CI job reporting its status.